Documentation/ITK5MigrationGuide.md

```diff
@@ -212,7 +212,7 @@ An external module example that demonstrates this can be found in
 [this commit](https://github.com/InsightSoftwareConsortium/ITKBoneMorphometry/pull/32/commits/a8014c186ac53837362a0cb9db46ae224b8e9584).
 
 Before, using `itk::Array`:
-```C++
+```cpp
```

Comment:
I thought that `C++` is a valid language specification for markdown. Is this not the case? Or are you just trying to make all instances of language specification uniform?

Reply:
Yes, `c++` works too.  I was going for uniformity, but it doesn't matter to me which one we use.  If you prefer the other way, I will do that.

Reply:
No strong preference.